host/default.yaml

```diff
@@ -352,20 +352,6 @@ spec:
         collectorName: "ps-high-load"
         command: "sh"
         args: ["-c", "ps -eo s,user,cmd | grep ^[RD] | sort | uniq -c | sort -nbr | head -20"]
-    - filesystemPerformance:
-        collectorName: filesystem-latency-two-minute-benchmark
-        timeout: 2m
```

Comment:
do we keep it but just reduce the timeout way down? 20-30sec instead of 2min?

Reply:
I like that, I have no idea why we would _need_ 2 minutes.